ODLA/platforms/vodla/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+add_definitions(-DTIMING)
+add_definitions(-DLOOP_CNT=1)
```

Comment:
what is LOOP_CNT used? for testing?

Reply:
LOOP_CNT is the number of vodh_infer called to evaluate the average inference time in remote container. LOOP_CNT should set to one in release version.